Formula/kubecfg.rb

```diff
@@ -0,0 +1,48 @@
+class Kubecfg < Formula
+  desc "Manage complex enterprise Kubernetes environments as code"
+  homepage "https://github.com/ksonnet/kubecfg"
+  url "https://github.com/ksonnet/kubecfg/archive/v0.7.2.tar.gz"
+  sha256 "a01de014904af9177bb0ad15a249346d4b7bd6412268d0f83756541aa19fa38c"
+  head "https://github.com/ksonnet/kubecfg.git"
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build if build.head?
+
+  def install
+    # Standard gopath shenanigans to keep "go build" happy:
+    sources = buildpath.children - [buildpath/".brew_home"]
+    ENV["GOPATH"] = buildpath
+    srcdir = buildpath/"src/github.com/ksonnet/kubecfg"
+    srcdir.mkpath
+    mv sources, srcdir
+    # The real build steps:
+    cd srcdir do
+      args = []
+      args.push("VERSION=#{version}") unless build.head?
+      system "make", "generate", *args if build.head?
+      system "make", *args
+      system "make", "test", *args if build.head?
+    end
+    # The install steps:
+    bin.install srcdir/"kubecfg"
+    sharelib = share/"lib"
+    sharelib.mkpath
+    sharelib.install srcdir/"lib/kubecfg.libsonnet"
```

Comment:
Note that the next and future releases (from https://github.com/ksonnet/kubecfg/pull/197) won't need this file to be separately installed - it now embeds kubecfg.libsonnet in the binary.

Current (and past) releases _do_ need this separately installed, however.

Edit: Ah! I see from the `caveats` text below, that you're already aware of this, and just installing it as a reference for the user.  Nice move.

Reply:
Ah, I thought it was already embedded in 0.7.2.

Reply:
I included a tiny patch to inject the `share/lib` directory at the end of the search path. `import "kubecfg.jsonnet"` now looks in the right place. Obviously it should go on the next release with the internalfs stuff obviating it.

Reply:
Is this still needed for `0.8.0`?

Reply:
No, but it's used in the `brew test` block.